src/tests/bananabot_test.py

```diff
@@ -0,0 +1,107 @@
+from bananabot import *
+
+class TestDefaultConfig:
+    def test_default_announcements_exist(self):
+        assert "banana_time" in announcements # ID of banana time announcement
+        assert 1 in announcements # ID of 10:00 announcement
+        assert 2 in announcements # ID of 60 mins before announcement
+        assert 3 in announcements # ID of 30 mins before announcement
+        assert 4 in announcements # ID of 10 mins before announcement
+
+    def test_default_banana_time(self):
+        assert "banana_time" in announcements
+        assert Announcement.banana_time == datetime.time(15, 30, 0)
+        assert announcements["banana_time"].time == datetime.time(15, 30, 0)
+        assert announcements["banana_time"].text == "# @HERE Banana Time!"
+
+    def test_default_active_value(self):
+        assert active == False
+
+
+class TestFunctions:
+    def test_start(self, mocker):
+        mocker.patch('bananabot.workers', {"1": 1, "2": 2})
+
+        assert start() == False
+
+    def test_stop(self, mocker):
+        mocker.patch('bananabot.workers', {})
+
+        assert stop() == False
+
+    def test_string_to_time(self):
+        result = string_to_time("15:30")
+        assert type(result) == datetime.time
+
+    def test_set_banana_time(self):
+        assert Announcement.banana_time == datetime.time(15, 30, 0)
```

Comment:
Be careful when unit testing against class variables. Depending on the test it may be useful to create setUp() or tearDown() methods to correctly restore the state of them prior to each test case.

Reply:
Good thinking. I will look into using some kind of setup and/or teardown method or modify the test so that this is not a concern.

Reply:
As it was just the one test I opted to just set `Announcement.banana_time` prior to checking it's value